### PR TITLE
Update Readme - Added table of contents and Component Properties

### PR DESCRIPTION
**Pull request by martypowell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
-# BCPL Blog
+## BCPL Blog
 
 Home of the code for the BCPL Blog.
 
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+- [BCPL Blog](#bcpl-blog)
```
Comment (jsmith176) [inline README.md]: Shouldn't the table of contents be at the start of the file?

File: README.md [hunk 2]
```diff
@@ -15,10 +20,19 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
 - Local - your web service running locally
 - Dev - testservices.bcpl.info
 - Prod - services.bcpl.info
+
+## Component Properties
+
+| Name                 | Type   | Default | Description                                                             | Is Required? |
+| -------------------- | ------ | ------- | ----------------------------------------------------------------------- | ------------ |
+| baseUrl              | string | n/a     | Api Endpoint for the Blog                                               | Yes          |
+| filters              | array  | n/a     | A list of filters to be applied to allow your users to use in their app | No           |
```
Comment (jsmith176) [inline README.md]: The description for filters is a tough read. Maybe try "A list of filters that may be applied to the blog".

File: README.md (hunk 1 above)
Comment (martypowell) [reply README.md]: yea i thought so too, but it looked funky, ill do a little more research

File: README.md (hunk 2 above)
Comment (ascire) [reply README.md]: Hey man, users use.

File: README.md (hunk 1 above)
Comment (martypowell) [reply README.md]: we agreed over chat, this is ok for now.

Comment (martypowell) [reply README.md]: ready for another look

File: README.md [hunk 3]
```diff
@@ -15,10 +22,36 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
+| cardContentComponent | Node   | n/a     | A custom component to display the results of your endpoint call | Yes          |
+| title                | string | ''      | H1 to show in your app                                          | No           |
+
+## Models
+
+### Filter
+
+**field**: property name of the filter in which you want to add (property must exist in your results)
+
+**type**: right now, this can only be `radio` (`checkbox` will be added in the future)
```
[the remark this answers is not in the document]
Comment (martypowell) [reply README.md]: good point, will do.

File: README.md [hunk 4]
```diff
@@ -15,10 +22,36 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
 - Prod - services.bcpl.info
+
+## Component Properties
+
+| Name                 | Type   | Default | Description                                                     | Is Required? |
+| -------------------- | ------ | ------- | --------------------------------------------------------------- | ------------ |
+| baseUrl              | string | n/a     | Api Endpoint for the Blog                                       | Yes          |
+| filters              | array  | n/a     | A list of [filters](#filter) to include in the app              | No           |
+| cardContentComponent | Node   | n/a     | A custom component to display the results of your endpoint call | Yes          |
+| title                | string | ''      | H1 to show in your app                                          | No           |
```
[the remark this answers is not in the document]
Comment (martypowell) [reply README.md]: Yea maybe it should be Headline? also the audience would be a developer.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,15 @@
-# BCPL Blog
+## BCPL Blog
```
[the remark this answers is not in the document]
Comment (martypowell) [reply README.md]: Yea, I had it and switched it, my Table of Contents generates automatically and it felt weird to me.

**Pull request merged**